glossary.md

```diff
@@ -19,4 +19,13 @@
 **Service provider**: The entity using Lock Keeper to build a full-featured digital asset manager.
 
 
+## Technical Definitions
+
+**Workflow/Flow**: A set of potentially non-linear steps that accomplish an action on behalf of a user. Completing a workflow requires the execution of one or more protocols. *Example: Generate and store a secret*
+
+**Protocol**: A multi-message cryptographic process that accomplishes a particular task within a workflow. *Example: Generate a secret*
+
+**Session**: An open channel of communication between a client and a server. Used to facilitate the execution of protocols or workflows.
+
+**Endpoint**: An RPC method that establishes a session and determines which protocol or workflow will be executed within that session.
```

Comment:
I meant to say `gRPC` here since that's the protocol we're using. I updated it to reflect that. I don't think it counts as an acronym since it's more of a named product. Maybe we could have a separate definition where we explain that and link to the gRPC website.

Reply:
I am happy with how you rephrased this.